src/helpers.php

```diff
@@ -30,6 +30,45 @@ function backpack_authentication_column()
     }
 }
 
+if (! function_exists('backpack_input_parse')) {
+    /**
+     * Parse the form submited input as it comes from request.
+     * Joins the multiple[] fields as a single key.
+     *
+     * Parse the HasOne relation names like address[street] to dot notation
+     *
+     * @return array
+     */
+    function backpack_input_parse($input)
+    {
```

Comment:
```suggestion
    {
        $input = $input ?? request('form');
```

Can/Should we ease this and assume most of the times developers are going to use `request('form')`?

Reply:
I think we could, but in that case we should rename the function to `backpack_form_input_parse()` and assume that the parsed input is always a form request. 

Do you foresee any scenario where we could use this to parse other inputs that are not in `request('form')` ? Otherwise we will endup `request()->merge('form',$data)` to call `backpack_form_input_parse()` and have it in the request.

Let me know,
Pedro

Reply:
_(if we are keeping the helper function)_
I agree with the name change, and maybe we can go even further 😅 `backpack_form_input()` parse is implicit.
If it's parsed or not, developers just want to know they are obtaining the form input data ready to use.

---

> Do you foresee any scenario where we could use this to parse other inputs that are not in request('form') ? Otherwise we will endup request()->merge('form',$data) to call backpack_form_input_parse() and have it in the request.

So, firstly yes, I can't see how will anyone need this using anything other than the `request('form')`.
Do you suggest to merge/replace the data on the request?
Can/Should we do that by default?